modules/lead/product-base/variables.tf

```diff
@@ -3,3 +3,6 @@ variable "product_name" {}
 variable "image_whitelist" {}
 variable "vault_namespace" {}
 variable "vault_root_token_secret" {}
+variable "vault_path_url" {
```

Comment:
I think `vault_address` would be a better name for this, since that's how it's used

Reply:
This is a much better name, so I've renamed it and I'll mark this as resolved.